Content.Client/Chemistry/Components/SmokeSystem.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Linq;
+using Content.Shared.Chemistry.Components;
+using Robust.Client.GameObjects;
+using Robust.Shared.GameStates;
+
+namespace Content.Client.Chemistry.Components;
+
+public sealed class SmokeSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<SmokeComponent, ComponentHandleState>(OnBlobTileHandleState);
+    }
+
+    private void OnBlobTileHandleState(EntityUid uid, SmokeComponent component, ref ComponentHandleState args)
```

Comment:
This isn't a blob tile. Stop copy-pasting shit blindly, deduplicate your code.

Reply:
oopse